jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobViewImpl.java

```diff
@@ -201,7 +204,15 @@ public void init() {
         initGridColumns();
         initDateTimePicker();
 
+        Button button = new Button(Constants.INSTANCE.Add_Parameter(), new ClickHandler() {
+            @Override
+            public void onClick(ClickEvent event) {
+                addNewParameter();
+            }
+        });
+        button.addStyleName("btn btn-default");
```

Comment:
Setting the style isn't needed for this component as this is the default already.